LayoutTests/dom/xsl/lockdown-mode/XSLT-disabled.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../../resources/js-test-pre.js"></script>
```

Comment:
```suggestion
<script src="../../../resources/js-test.js"></script>
```

The pre/post variants are obsolete.

Reply:
Ah, ace, great to know - thank you! Pushed new change with this corrected.